packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-flow.tsx

```diff
@@ -44,13 +36,7 @@ const ProofOfAddressFlow = observer(() => {
         return url.toString();
     };
 
-    if (isRedirectToAccountsOSAppFFLoaded && !isKYCLoading) {
-        if (shouldRedirectToAccountsOSApp && kyc_auth_status) {
-            window.location.replace(getFormattedURL(ACCOUNTS_OS_POA_URL));
-        } else {
-            return <ProofOfAddress />;
-        }
-    }
+    window.location.replace(getFormattedURL(ACCOUNTS_OS_POA_URL));
```

Comment:
what about POI?

Reply:
I mentioned in the description that the main purpose of this PR is: Removing the POA feature flag to terminate Deriv Verify traffic